app/controllers/oai_controller.rb

```diff
@@ -1,34 +1,47 @@
 # frozen_string_literal: true
 
 # Produce OAI-PMH responses
+# rubocop:disable Metrics/ClassLength
 class OaiController < ApplicationController
   load_and_authorize_resource :organization
 
-  before_action do
-    @stream = @organization.default_stream
-    authorize! :read, @stream
-  end
-
   def show
```

Comment:
maybe this should be replaced with something, I'm just unfamiliar with how authorization works in controllers. should any authenticated user be able to `ListSets`?

Reply:
It seems ok to allow any authenticated user to make a ListSets request.